test/dlc_tests/sanity/test_security_check.py

```diff
@@ -21,3 +27,38 @@ def test_security(image):
         run(f"{docker_exec_cmd} python /test/bin/security_checks.py ")
     finally:
         run(f"docker rm -f {container_name}", hide=True)
+
+
+@pytest.mark.model("N/A")
```

Comment:
let's mark as canary - are there any limits we could run into?

Reply:
Yes, I'll mark this as canary. I don't think there are any limits we would run into. The scan itself can only be run once per day. If we run multiple scans on the same image in a period of 24 hours, all the scans after the first will fail with a `LimitExceededException`.

This test is written in such a way that it can "skip" the scan if there is a `LimitExceededException`.